src/index.html

```diff
@@ -16,7 +37,496 @@
+                      src="images/page/old-masters-02.png"
+                      alt="Black man using collar"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+                <h4 class="category__title">A New Look at Old Masters</h4>
+                <p>December 12, 2020 – Through Spring 2023</p>
+              </article>
+              <article class="category">
+                <div class="category__photos category__photos--middle">
+                  <a
+                    href="#"
+                    class="category__link category__link--square"
+                  >
+                    <img
+                      src="images/page/bodhisattvas-01.png"
+                      alt="new-look"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link category__link--square"
+                  >
+                    <img
+                      src="images/page/bodhisattvas-02.png"
+                      alt="new-look"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link category__link--square"
+                  >
+                    <img
+                      src="images/page/bodhisattvas-03.png"
+                      alt="new-look"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+                <h4 class="category__title">
+                  Bodhisattvas of Wisdom, Compassion, and Power
+                </h4>
+                <p>March 27, 2021 – October 30, 2022</p>
+              </article>
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link category__link--square"
+                  >
+                    <img
+                      src="images/page/french-decorative-arts-01.png"
+                      alt="new-look"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link category__link--wide"
+                  >
+                    <img
+                      src="images/page/french-decorative-arts-02.png"
+                      alt="new-look"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+                <h4 class="category__title">
+                  Inspiring Walt Disney: The Animation of French Decorative Arts
+                </h4>
+                <p>December 10, 2021 – March 6, 2022</p>
+              </article>
+            </div>
+          </section>
+          <!-- explore-online -->
```

Comment:
You're missing a button here:
Your page:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/a73ea16c-f189-445b-8e6c-f69af307af1c)

Figma:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/76d52395-eaad-4cb0-b0a6-76da0b2ba852)

Reply:
Done